src/app/forms/templates/test-records/master.template.ts

```diff
@@ -261,6 +264,26 @@ export const masterTpl: Record<VehicleTypes, Partial<Record<keyof typeof TEST_TY
       reasonForCreation: reasonForCreationSection,
       required: SpecialistRequiredSectionHGVTRL,
       customDefects: CustomDefectsSection
+    },
+    testTypesDeskBasedGroup1: {
+      required: RequiredSection,
+      vehicle: DeskBasedVehicleSectionHgvGroup124,
```

Comment:
Whats 124 like 1 to 4 ?

Reply:
Groups 1, 2 and 4

Reply:
should it be name 1And2And4 then? keep it consistent with other templates?